<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AlgorithmRunner</class>
 <widget class="QWidget" name="AlgorithmRunner">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>384</width>
    <height>710</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>0</height>
   </size>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="HeaderFrame" name="header" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title" stdset="0">
      <string>Algorithms</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="ClearableLineEdit" name="searchBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>22</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>22</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#searchBox {
  border-left: 1px solid &quot;#C9C9C9&quot;;
  border-right: 0px;
  border-top: 0px;
  border-bottom: 1px solid &quot;#C9C9C9&quot;;
}</string>
     </property>
     <property name="placeholderText">
      <string>Filter name...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="handleWidth">
      <number>1</number>
     </property>
     <widget class="QScrollArea" name="algorithmList">
  <property name="styleSheet">
   <string notr="true">
ExpandableGroupBox[ expanded = &quot;true&quot; ] {
margin-top: 1.5ex;
}
ExpandableGroupBox[ expanded = &quot;false&quot; ] {
margin-top: 0ex;
margin-left: 0ex;
}
ExpandableGroupBox::title {
subcontrol-origin: margin;
subcontrol-position: top left;
padding: 0 3px;
}
ExpandableGroupBox::indicator:unchecked {
image: url(:/tulip/gui/ui/right_arrow-black.png);
}
ExpandableGroupBox::indicator:checked {
image: url(:/tulip/gui/ui/down_arrow-black.png);
}
ExpandableGroupBox {
border: 1px solid #E9E9E9;
}
ExpandableGroupBox[ root = &quot;true&quot; ] {
font: bold;
}
ExpandableGroupBox[ root = &quot;false&quot; ] {
border: 0px;
}
ExpandableGroupBox[root = &quot;true&quot; ][expanded = &quot;true&quot; ] {
margin-left: 1ex;
}
ExpandableGroupBox[root = &quot;false&quot; ][expanded = &quot;true&quot; ] {
margin-left: 1ex;
}
</string>
  </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="contents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>384</width>
         <height>678</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="contentsLayout">
          <property name="spacing">
           <number>10</number>
          </property>
          <property name="leftMargin">
           <number>3</number>
          </property>
          <property name="topMargin">
           <number>6</number>
          </property>
          <property name="rightMargin">
           <number>3</number>
          </property>
          <property name="bottomMargin">
           <number>6</number>
          </property>
          <item>
           <widget class="FavoriteBox" name="favoritesBox">
            <property name="styleSheet">
             <string notr="true">#favoritesBox {
border: 0px;
}

#favoritesBox::title {
spacing: 24px;
}</string>
            </property>
            <property name="title">
             <string>Favorites</string>
            </property>
            <property name="root" stdset="0">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HeaderFrame</class>
   <extends>QWidget</extends>
   <header>HeaderFrame.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ClearableLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">tulip/ClearableLineEdit.h</header>
  </customwidget>
  <customwidget>
   <class>FavoriteBox</class>
   <extends>QGroupBox</extends>
   <header>ExpandableGroupBox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>searchBox</sender>
   <signal>textChanged(QString)</signal>
   <receiver>AlgorithmRunner</receiver>
   <slot>setFilter(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>192</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>192</x>
     <y>355</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setFilter(QString)</slot>
 </slots>
</ui>
